src/editors/data/redux/app/selectors.js

```diff
@@ -1,4 +1,9 @@
 import { createSelector } from 'reselect';
+import { getEditorData,
```

Comment:
note that if you are doing operations to the problem editor state, they probably won't end up in app selectors, so it might be worth while to get rid of this import.